<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AggregationWidget</class>
 <widget class="QWidget" name="AggregationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="enabledCheckBox">
     <property name="toolTip">
      <string>Check to set aggregation period</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="longLabel">
     <property name="text">
      <string>Interval of action :</string>
     </property>
    </widget>
   </item>
   <item>
     <widget class="QLabel" name="everyLabel">
       <property name="text">
         <string>No more than once per</string>
       </property>
     </widget>
   </item>
   <item>
     <widget class="nx::client::desktop::TimeDurationWidget" name="periodWidget" native="true">
     </widget>
   </item>
   <item>
    <widget class="QLabel" name="instantLabel">
     <property name="text">
      <string>Instant</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
  <customwidgets>
    <customwidget>
      <class>nx::client::desktop::TimeDurationWidget</class>
      <extends>QWidget</extends>
      <header location="global">ui/widgets/business/time_duration_widget.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
 <resources/>
 <connections/>
</ui>
